process_list.ipynb

```diff
@@ -19,8 +19,62 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import ipywidgets as ipw\n",
     "from aiidalab_widgets_base import ProcessListWidget\n",
-    "p = ProcessListWidget()"
+    "from traitlets import dlink\n",
+    "from plumpy import ProcessState\n",
+    "\n",
+    "process_list = ProcessListWidget()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "past_days_widget = ipw.IntText(value=7, description='Past days:')\n",
+    "dlink((past_days_widget, 'value'), (process_list, 'past_days'))\n",
+    "\n",
+    "\n",
+    "all_days_checkbox = ipw.Checkbox(description=\"All days\", value=False)\n",
+    "dlink((all_days_checkbox, 'value'), (past_days_widget, 'disabled'))\n",
+    "dlink((all_days_checkbox, 'value'), (process_list, 'past_days'), transform=lambda v: -1 if v else past_days_widget.value)\n",
```

Comment:
I think you might need to `observe` the `all_days_checkbox` instead, because you probably also want to disable the `past_days_widget` when the `all_days_checkbox` is selected, right?

Reply:
That is what I do on the line above, right?
```python
dlink((all_days_checkbox, 'value'), (past_days_widget, 'disabled'))
```

Reply:
Sorry, I overlooked that.